website/source/docs/providers/aws/r/route.html.markdown

```diff
@@ -35,12 +35,14 @@ The following arguments are supported:
 * `destination_cidr_block` - (Required) The destination CIDR block.
 * `vpc_peering_connection_id` - (Optional) An ID of a VPC peering connection.
 * `gateway_id` - (Optional) An ID of a VPC internet gateway or a virtual private gateway.
```

Comment:
I think the relevant part of the Go SDK is [here](https://github.com/aws/aws-sdk-go/blob/18177cad2597e3cac4173b4f267b784abdb3c854/service/ec2/api.go#L1370), which at first glance does not require specifying the type of gateway. However, I'm still reading over it and am new to Go. Let me know what I'm missing.